### ISSUE TITLE
Unable to silence flymake

### ISSUE THREAD

**Issue by memeplex** [jorgenschaefer/elpy#1386]
As is customary in my ipython config file I have a `get_config()` reference to a non imported symbol. This is not an error and I want to silence the linter using `# noqa`.

Now, this isn't working but I think it's because two different checkers are being run. `flake8` is indeed honouring `# noqa` but the second checker is not.

My supposition about the second checker is based on:

1. Without `#noqa` the error is reported twice, one of the reports include the flake8 code the other just the message.
2.  With `#noqa` the error is reported once without the flake8 code.
3. Style/convention warnings are completely suppresed by `#noqa`.

**Comment by galaunay** [comment]
I guess you are using the brand new emacs 26.1.

From emacs 26.1, python.el supports flymake, so Elpy's flymake module is kind of redundant.

At the moment, I would guess you see the results from elpy (that uses `flake8`) and also from python.el (that uses `pyflakes` by default).

As a temporary workaround, I would suggest deactivating elpy's flymake module through customize, or with:
```Elisp
(setq elpy-modules (delete 'elpy-module-yasnippet elpy-modules))
```

or removing python support for flymake:
```Elisp
(setq flymake-diagnostic-functions '(t))
```

**Comment by memeplex** [comment]
Thanks!

Have you decided whether you are gonna remove flymake support altogether from elpy and depend on the one in python.el or to disable this last one?

**Comment by memeplex** [comment]
I assume you meant

```
(setq elpy-modules (delete 'elpy-module-flymake elpy-modules))
```

Nevertheless, after doing that I get not flymake at all!

**Comment by memeplex** [comment]
Doing this I still get the error twice reported:

```
(elpy-enable)
(setq elpy-modules (delete 'elpy-module-flymake elpy-modules))
(add-hook 'python-mode-hook 'flymake-mode)
```

**Comment by memeplex** [comment]
Ok, I had to use customize in order to declaratively remove the module and *after that* enabling elpy.

**Comment by galaunay** [comment]
Ok, thanks for the feedback.
I hope it will be fixed by #1387.
